### ISSUE TITLE
Accessibility 

### ISSUE THREAD

**Issue by shazahm1** [shazahm1/Easy-Table-of-Contents#74]
- Heading: presently, you use a p element for the "Table of Content" heading. This is a true heading and should be marked up that way semantically. Consider an h2 heading (or allowance to customize in the settings).
- nav: While there is a nav element, it's generic, meaning, it is not labeled (especially important, as there are multiple navs on the page). Screen reader users will then have the context of the nav region announced. This can easily be done by adding an ID on the "Table of Content" heading (see above), then reference this ID on the nav with aria-labelledby.
- Expand/Collapse button: The ToC Expand/Collapse button is not keyboard accessible, nor does it provide state (collapsed, expanded). You are using an anchor element a for the button. Consider using a button element for this control (while will give you the keyboard accessibility, or, if you still wish to use the anchor, add role="button" and tabindex="0" to the a element. Lastly, use the aria-expanded="true/false" attribute on this control (dynamically updated via scripting) to communicate whether the ToC is expanded or collapsed.
